.github/workflows/build.yml

```diff
@@ -6,6 +6,9 @@ on:
       - '!refs/tags/.*'
     tags-ignore:
       - '*'
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Maybe the same tag and branch rules should be added for symmetry

Reply:
I have no opinion.

- For Nextflow they are not added: https://github.com/nextflow-io/nextflow/blob/master/.github/workflows/build.yml
- For Wave only the `paths_ignore` are added: https://github.com/seqeralabs/wave/blob/master/.github/workflows/build.yml

Reply:
I mean for symmetry with the `push` section in this build. Added

Reply:
I know - what I meant is that in the other 2 projects there were tag/branch rules for pushes, which were not mirrored for PRs